modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/category/facet/configuration.jsp

```diff
@@ -79,6 +84,28 @@ CategoryFacetPortletPreferences categoryFacetPortletPreferences = new CategoryFa
 				<aui:input label="frequency-threshold" name="<%= PortletPreferencesJspUtil.getInputName(CategoryFacetPortletPreferences.PREFERENCE_KEY_FREQUENCY_THRESHOLD) %>" value="<%= categoryFacetPortletPreferences.getFrequencyThreshold() %>" />
 
 				<aui:input label="display-frequencies" name="<%= PortletPreferencesJspUtil.getInputName(CategoryFacetPortletPreferences.PREFERENCE_KEY_FREQUENCIES_VISIBLE) %>" type="checkbox" value="<%= categoryFacetPortletPreferences.isFrequenciesVisible() %>" />
+
+				<c:choose>
+					<c:when test="<%= assetCategoriesSearchFacetDisplayContext.isDisplayVocabularySelector() %>">
+						<div id="<portlet:namespace />selectVocabularies">
+							<react:component
+								module="js/components/SelectVocabularies"
+								props='<%=
+									HashMapBuilder.<String, Object>put(
+										"initialSelectedVocabularyIds", StringUtil.merge(categoryFacetPortletPreferences.getVocabularyIds())
+									).put(
+										"namespace", liferayPortletResponse.getNamespace()
+									).put(
+										"vocabularyIdsInputName", PortletPreferencesJspUtil.getInputName(CategoryFacetPortletPreferences.PREFERENCE_VOCABULARY_IDS)
+									).build()
+								%>'
+							/>
+						</div>
+					</c:when>
+					<c:otherwise>
+						render the selector "disabled" and add help text/"Learn more" link
```

Comment:
@thektan @oliv-yu 
can one of you guys help with this? more context if you need it, point 2 of tibor's comment:
https://liferay.slack.com/archives/C03LFD93BPF/p1663668159684479?thread_ts=1662733042.084449&cid=C03LFD93BPF

Reply:
@BryanEngler sure, I can work on it!

Reply:
@lipusz @rbohl Just checking, which help text are we using for this disabled part?

Reply:
@oliv-yu as far as i know, no one has come up with what the text should say yet... ill think about it a little bit

Reply:
im thinking just something simple like this?

> To enable this configuration please visit the documentation and follow the instructions. "Learn more"

Reply:
Bryan's text is good. I'd only suggest that we can make it shorter
> To enable this configuration follow the steps in the documentation. Learn More.